TcUnit-Runner/VisualStudioInstance.cs

```diff
@@ -328,12 +328,16 @@ public EnvDTE80.DTE2 GetDevelopmentToolsEnvironment()
 
         public void CleanSolution()
         {
-            visualStudioSolution.SolutionBuild.Clean(true);
+            visualStudioSolution.SolutionBuild.Build(false);
```

Comment:
Why does the cleansolution() do a build()?

Reply:
I was in a hurry, sorry